db/migrations/00024_create_storage_diffs_table.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+CREATE TABLE public.storage_diff
+(
+    id             SERIAL PRIMARY KEY,
```

Comment:
Should copy over the new migration from vulcanizedb that makes this a `bigserial`

Reply:
nice catch! will do.